ag_data/simulator.py

```diff
@@ -1,146 +0,0 @@
-from time import sleep
-from random import gauss, random
-
-from django.utils import timezone
-
-from ag_data.formulas.ingestion_engine import (
-    MeasurementIngestionEngine,
-    MeasurementExchange,
-)
-from ag_data import utilities
-
-
-class Simulator:
```

Comment:
I agree that the simulator is not a good way to do this, but could you give an example of how we will be generating measurements for debugging and manual testing the measurement pipeline, now that we are going to have the simulator?

Reply:
1.
As of now we cannot generate a series of measurements over time and it's true that we lose that functionality.

2.
We can build a new one using the simulator script (`radioport.py`). That would be more meaningful since fake data follow the similar path which one from real h/w do.

3.
Also, our tests cover measurement pipeline and personally I don't see any case that we need to do the test manually except integration test.
When it comes to integration test, you know the old simulator.py doesn't fulfill our needs.
Also, we can run our django app on heroku and can test it manually. But in this case, we don't still need simulator.py.

Reply:
When we test against synchronization, it is necessary to have some ways to easily generate data. So I think we need some sort of that tools at least for the next two weeks or so. Otherwise, I think it will be helpful to provide a sample way for people to generate that. Or you could share how you generate data during testing?

Reply:
We can add measurements in django admin. That worked for me when I worked on synchronization.

Reply:
I get what you mean. I rarely use the admin page, because it requires the server to be running, and it does not suit certain situations. But I can deal with it for now. And for the demo this week, let's just use the admin portal. 

For this particular discussion, I am okay to the idea to keep the design this PR presents. I do think we should update the radio simulator script next sprint, to enable this sort of functionality, and also help with integration tests. Does this seem like a workable plan for you?

Reply:
Yes, works for me 👍